pkgs/development/interpreters/lua-5/build-luarocks-package.nix

```diff
@@ -196,6 +199,7 @@ let
     # maybe we could reestablish dependency checking via passing --rock-trees
 
     nix_debug "ROCKSPEC $rockspecFilename"
+    # deps-mode=all tells luarocks to use every configured rocks_trees
```

Comment:
question: this looks like it's left over from something else?

Reply:
I sneaked that in since I thought it would be helpful for contributors. True it's unrelated I can push it in a different PR but it seemed hartless and it goes to staging so rebuilds are not an issue. Rest of PR triggers rebuild too.